partiql-ast/src/main/java/org/partiql/ast/v1/DataType.java

```diff
@@ -1,6 +1,7 @@
 package org.partiql.ast.v1;
 
 import lombok.EqualsAndHashCode;
+import org.jetbrains.annotations.NotNull;
 
 @EqualsAndHashCode(callSuper = false)
 public class DataType implements Enum {
```

Comment:
for every AST class implementing the `enum` interface, added the `valueOf(String value)` and `values()` static methods